eks/cluster-loader/cluster-loader-add-on/cluster-loader.gotmpl

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: clusterloader2
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: clusterloader2
+  namespace: clusterloader2
+automountServiceAccountToken: true
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: clusterloader2-cluster-admin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    name: clusterloader2
+    namespace: clusterloader2
+
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: clusterloader2
+  namespace: clusterloader2
+spec:
+  template:
+    spec:
+      containers:
+      - name: clusterloader2
+        image: {{ .Image }}
+        command: ["/clusterloader"]
+        args: {{ .TestParams }}
```

Comment:
I'd probably pass this in the inline struct.

Reply:
Got it.